imports/api/inventories.js

```diff
@@ -41,7 +41,12 @@ Meteor.methods({
 		}
 
 		if (amount.length === 0) {
-			throw new Meteor.Error('amount cannot be empty')
+			throw new Meteor.Error('Amount cannot be empty')
+		}
+
+		let inventory = Inventories.findOne(itemId);
+		if (inventory.owner !== this.userId) {
```

Comment:
Cool 🙂once we merge my changes, we can use the `authCheck` function that I built to check authorization-related things like this

Reply:
Yep, that's the plan